goal_time_1: '02:00'
goal_time_10: '18:15'
goal_time_2: '18:00'
goal_time_3: '18:05'
goal_time_4: '18:07'
goal_time_5: '18:10'
goal_time_6: '12:08'
goal_time_7: '05:00'
goal_time_8: '18:12'
goal_time_9: '18:17'
